scrc/preprocessors/extractors/spider_specific/section_splitting_functions.py

```diff
@@ -605,6 +610,8 @@ def get_paragraphs(content):
                     else:
                         paragraph = text
                     heading = None  # reset heading
+                if paragraph not in ['', ' ', None]:  # only clean non-empty paragraphs
+                    paragraph = clean_text(paragraph)
                 if paragraph not in ['', ' ', None]:  # discard empty paragraphs
                     paragraphs.append(paragraph)
```

Comment:
I think line 614 can be just after the second if statement. Then you don't need two ifs, right?

Reply:
Yes, I have now combined the two if statements.